docs/modules/opa/pages/index.adoc

```diff
@@ -1,18 +1,37 @@
 = Stackable Operator for OPA (OpenPolicyAgent)
+:description: The Stackable Operator for OPA is a Kubernetes operator that manages OPA instances. Learn about its features, resources, dependencies and demos, and see the list of supported OPA versions.
+:keywords: Stackable Operator, OPA, OpenPolicyAgent, Rego, Kubernetes, k8s, operator, engineer, policy
 
-This is an operator for Kubernetes that can manage https://www.openpolicyagent.org/[OPA] servers.
+The Stackable Operator for the https://www.openpolicyagent.org/[OpenPolicyAgent] (OPA) manages OPA instances. OPA is an open-source policy engine that allows you to define, manage and enforce policies across a number of software systems. OPA promotes "policy-as-code"; policies are defined in the declarative https://www.openpolicyagent.org/docs/latest/policy-language/[Rego] language.
 
-WARNING: This operator only works with images from the https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%2Fopa[Stackable] repository
+== Getting started
 
-== Supported Versions
+The xref:getting_started/index.adoc[Getting started guide] will guide you through the installation of the Operator and setting up OPA. You will also create your first Rego rule and query the OPA for a policy decision.
 
-The Stackable Operator for OPA currently supports the following versions of OPA:
+== Operator model
 
-include::partial$supported-versions.adoc[]
+The _OpaCluster_ custom resource is used to declare OPA instances, only one xref:home:concepts:roles-and-role-groups.adoc[role] is defined: `server`. The biggest difference to other products on the Stackable Data Platform is that OPA is deployed as a DaemonSet (not as a StatefulSet). Policy decisions need to be made quick, so it is helpful to have an agent running on every node, to avoid network calls. For every role group a DaemonSet is created, with its own ConfigMap. The DaemonSet will then create a Pod for every node. Every role group also gets its own Service definition.
```

Comment:
I removed the DaemonSet vs StatefulSet comparison for future changes. We will probably move away from statefulsets to deployments in a couple of products.
```suggestion
The _OpaCluster_ custom resource is used to declare OPA instances, only one xref:home:concepts:roles-and-role-groups.adoc[role] is defined: `server`. OPA is deployed as a DaemonSet because policy decisions must to be fast and efficient. Therefore an OPA agent must be available on every Node to reduce latency and network calls. A DaemonSet with its own ConfigMap is created for every role group. The DaemonSet will then deploy a Pod on every node. Every role group also gets its own Service definition.
```